src/system/RoutesModule/Api/Base/CacheApi.php

```diff
@@ -14,14 +14,14 @@
 
 use ModUtil;
 use UserUtil;
-use Zikula_AbstractApi;
+use Zikula_AbstractBase;
 use Zikula_View;
 use Zikula_View_Theme;
 
 /**
  * Cache api base class.
  */
-class CacheApi extends Zikula_AbstractApi
+class CacheApi extends Zikula_AbstractBase
```

Comment:
I think you are confusing legacy `Zikula_AbstractApi` and `Zikula\Core\Api\AbstractApi` which was [just removed](https://github.com/zikula/core/pull/2575). The legacy [Zikula_AbstractApi](https://github.com/zikula/core/blob/1.4/src/lib/legacy/Zikula/AbstractApi.php) class still exists.